src/pages/proposal/completed/GeneralVote.tsx

```
@@ -38,13 +42,31 @@
+              sanitizeOptions
+            ),
+          }}
+        ></Typography>
         <Typography
           variant="caption"
           className="clamp-3-lines"
           sx={{
             color: '#CBD5E1',
           }}
-          dangerouslySetInnerHTML={{ __html: description }}
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)